Updated Display class to contain a video type reference

JasperB-TeamBlue opened the pull request:
* Added VideoType as an extra parameter in the display, opens opportunity to alter/view the video type used by an object.
*  Added documentation to VmGraphicsConsolesService.

dupondje commented:
This is the replacement for https://github.com/oVirt/ovirt-engine-api-model/pull/102
@sgratch : might want to check if this is how you intended it or if you have any remarks

JasperB-TeamBlue committed:
api: updated Display class to contain a video type reference

Added VideoType as an extra parameter in the display, opens opportunity to alter/view the video type used by an object.
Added documentation to VmGraphicsConsolesService.

Signed-off-by: Jasper Berton <jasper.berton@team.blue>
